utils/api_info_v2.py

```diff
@@ -61,8 +63,20 @@
     # Go!
     r = requests.get(api_info_url, auth=auth)
 
-    #print(r.text)
-
-    # For printing unicode instead of unicode escape sequences
-    jr = json.loads(r.text)
-    pprint(jr)
+    if args.raw:
+        print(r.text)
+    else:
+        try:
+            rj = r.json()
+        except ValueError:
+            print('Bad response:')
+            print(r.text)
+            exit(1)
+
+        errors = rj.get('errors')
+
+        if errors:
+            print('Info request failed:',  errors)
+            exit(1)
+        else:
+            print("Torrent #{} '{}' uploaded by '{}' ({} bytes) {} / {}\n{}".format(rj['id'], rj['name'], rj['submitter'], rj['filesize'], rj['main_category'], rj['sub_category'], rj['magnet']))
```

Comment:
@shm0o 
I believe there's a `vformat` function (not sure about the name) so you can use it directly on the dictionary (no packing then unpacking)